extensions/iceberg-aws/src/main/java/io/deephaven/iceberg/util/IcebergToolsAWS.java

```diff
@@ -0,0 +1,64 @@
+//
+// Copyright (c) 2016-2024 Deephaven Data Labs and Patent Pending
+//
+package io.deephaven.iceberg.util;
+
+import com.google.common.base.Strings;
+import org.apache.iceberg.CatalogProperties;
+import org.apache.iceberg.CatalogUtil;
+import org.apache.iceberg.aws.AwsClientProperties;
+import org.apache.iceberg.aws.s3.S3FileIOProperties;
+import org.apache.iceberg.io.FileIO;
+import org.apache.iceberg.rest.RESTCatalog;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Tools for accessing tables in the Iceberg table format.
+ */
+public class IcebergToolsAWS extends IcebergTools {
```

Comment:
I suspect this should be `S3` not `AWS`, as it stands right now. Same for the module name.